tests/e2e/utils/asset.py

```diff
@@ -2,18 +2,46 @@
 import json
 import zipfile
 from pathlib import Path
-from typing import List, Optional
+from typing import Dict, List, Optional
 
 import boto3
 import botocore.exceptions
 from mypy_boto3_s3 import S3Client
+from pydantic import BaseModel, Field
 
 from aws_lambda_powertools import Logger
-from tests.e2e.utils.models import AssetTemplateConfig, TemplateAssembly
 
 logger = Logger(service="e2e-utils")
 
 
+class AssetTemplateConfigSource(BaseModel):
```

Comment:
I would rename AssetTemplateConfig to AssetManifest following official naming convention

Reply:
Great catch!